includes/data-stores/class-wcs-subscription-data-store-cpt.php

```diff
@@ -225,7 +225,7 @@ protected function update_post_meta( &$subscription ) {
 	 * @return string
 	 * @since 2.2.0
 	 */
-	public function get_total_refunded( $subscription ) {
+	public function get_total_refunded( $subscription ) : float {
```

Comment:
This is to fix fatal error `Declaration of WCS_Subscription_Data_Store_CPT::get_total_refunded($subscription) must be compatible with WC_Order_Data_Store_CPT::get_total_refunded($order): float`.